pkg/promtail/client/client.go

```diff
@@ -330,3 +367,11 @@ func (c *client) Handle(ls model.LabelSet, t time.Time, s string) error {
 	}}
 	return nil
 }
+
+func (c *client) RegisterLatencyMetric(labels model.LabelSet) {
```

Comment:
I guess you don't need this anymore.

Reply:
yup, i went down a few paths of making this metric more generic because I think it could also be useful with the loki push api, however i scaled back for time/complexity and we can add that functionality another day if we need it.